php/api/controller/class-audit-posts-controller.php

```diff
@@ -342,6 +344,31 @@ public function prepare_item_for_response( $post, $request ) {
 			$response->remove_link( $field );
 		}
 
+		// Only for authenticated users.
+		if ( User::authenticated() ) {
+
+			// Only interested in _audit_<standard> meta.
+			$meta = array_filter( get_post_meta( $post->ID ), function ( $k ) {
+				return strpos( $k, '_audit_' ) !== false;
+			}, ARRAY_FILTER_USE_KEY );
+
+			foreach ( array_keys( $meta ) as $standard ) {
```

Comment:
The `standards` array should have the information you need. Something along the lines of:
```
// Get "standards" from the post meta.
$standards = get_post_meta( $post->ID, 'standards', true );

// Filter using the allowed standards.
if ( ! empty( $standards ) ) {
    $allowed_standards = array_keys( Audit::allowed_standards() );

    $standards = array_filter( $standards, function ( $standard ) use ( $allowed_standards ) {
        return in_array( $standard, $allowed_standards, true );
    } );
} else {
    $standards = array_keys( Audit::executable_audit_fields() );
}
```